tests/extmod/uasyncio_threadsafeflag.py.exp

```diff
@@ -0,0 +1,21 @@
+yield
+task 1
+set event
+yield
+wait task
+task 1 done
+----
+yield
+task 2
+set event
+yield
+schedule
+schedule done
+wait task
+task 2 done
+----
+set event
+yield
+task 3
+wait task
+task 3 done
```

Comment:
when I run this test on a PYBD-SF6 I get:
```
set event
yield
task 3
task 3 done
wait task
```
which makes more sense to me than this .exp file, that task 3 starts and is immediately finished